### PR TITLE
Fix some popup style issues.

### LINKED ISSUE #7234: AddOnInstallationDialogFragment layout alignment issues
Some elements (ie the checkbox and "Allow in Private Browsing") in the dialog that shows after installing an addon aren't vertically aligned.

image

### PR DESCRIPTION
**Pull request by person808**
---

Addresses these layout issues: https://github.com/mozilla-mobile/fenix/issues/11023#issuecomment-640347274

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not

Just a layout change

- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

**Comment by person808** [comment]
The lint error should be fixed now. For point 2, the original fenix issue (mozilla-mobile/fenix#11023) was mentioned to me as a good first issue to try and fix. But I see that the other dialogs use the same alignment so I suppose it would have to be a broader UX discussion. I'm not familiar with who to talk to about that.

**Comment by person808** [comment]
@psymoon Updated with the ux feedback

File: components/feature/addons/src/main/res/layout/mozac_feature_addons_fragment_dialog_addon_installed.xml [hunk 1]
```diff
@@ -84,6 +85,8 @@
         android:layout_marginTop="16dp"
         android:layout_marginEnd="16dp"
         android:layout_marginBottom="16dp"
+        android:paddingStart="20dp"
```
Comment (psymoon) [inline components/feature/addons/src/main/res/layout/mozac_feature_addons_fragment_dialog_addon_installed.xml:88]: I think the padding here is 6 as per https://github.com/mozilla-mobile/fenix/issues/11023#issuecomment-640347274

Comment (person808) [reply components/feature/addons/src/main/res/layout/mozac_feature_addons_fragment_dialog_addon_installed.xml:88]: The comment says that the "Okay, Got it" button should match the padding of the "Allow" button which is 20dp. Unless I'm reading it wrong?

**Pull request merged**